packages/sdk/test/utils/generateRandomString.ts

```diff
@@ -0,0 +1,5 @@
+export const generateRandomString = (len = 40) => {
+	return Array(len + 1)
+		.join((Math.random().toString(36) + '00000000000000000').slice(2, 18))
```

Comment:
It's a poor random generator as it repeats the pattern of the first 16 chars, e.g.:
'01q9sg39xzxa000001q9sg39xzxa000001q9sg39'
'y4p7monromc00000y4p7monromc00000y4p7monr'
'yza3xn9l71h00000yza3xn9l71h00000yza3xn9l'

So, what's the point of the `len` parameter as it does not improve uniqueness? We can just generate a 16-length string and make a code simpler.